app/src/main/java/com/hyorim/sopt_assigmnet_1/BindingConversions.kt

```diff
@@ -4,13 +4,28 @@ import android.graphics.drawable.Drawable
 import android.widget.ImageView
 import androidx.databinding.BindingAdapter
 import com.bumptech.glide.Glide
+import com.bumptech.glide.annotation.GlideModule
+import com.bumptech.glide.module.AppGlideModule
 
 object BindingConversions {
 
-    @BindingAdapter("imageUrl")
+    /** follower_list 의 팔로워 프로필 */
+    @BindingAdapter("imageUrl")       // Binding Adapter 생성
+    @JvmStatic                                   // Static 함수로 설정해주기 위한 Annotation
+    fun loadImage(imageView : ImageView, url : String){
+        Glide.with(imageView.context)
+            .load(url)
+            .circleCrop()
+            .into(imageView)
+    }
```

Comment:
사실 BindingAdapter의 경우 확장함수의 형태로

```suggestion
    fun ImageView.loadImage(url : String){
        Glide.with(context)
            .load(url)
            .circleCrop()
            .into(this)
    }
```
이렇게도 표현가능~!!